packages/dts-test/setupHelpers.test-d.ts

```diff
@@ -501,3 +502,21 @@ describe('toRefs w/ type declaration', () => {
   }>()
   expectType<Ref<File | File[] | undefined>>(toRefs(props).file)
 })
+
+describe('defineOptions', () => {
+  defineOptions({
+    name: 'MyComponent',
+    inheritAttrs: true,
+  })
+
+  defineOptions({
+    // @ts-expect-error props should be defined via defineProps()
+    props: 'props',
```

Comment:
I think it should use proper `props` value - in case of introducing a bug that makes `defineOptions` to allow valid props this test will pass, as there will be the expected ts error caused by value not matching the props type.
`emits` and `slots` I think are fine here, `expose` afair expects and object, not sure

Reply:
👍 I forgot to wrap this one in an array to make it valid